packages/devextreme/js/__internal/ui/list/list.base.ts

```diff
@@ -57,9 +59,7 @@ import ScrollView from '@ts/ui/scroll_view/scroll_view';
 import { deviceDependentOptions } from '@ts/ui/scroll_view/scrollable.device';
 import type { ScrollOffset } from '@ts/ui/scroll_view/types';
 import { getElementMargin } from '@ts/ui/scroll_view/utils/get_element_style';
-import DataConverterMixin from '@ts/ui/shared/m_grouped_data_converter_mixin';
-
-import type { CollectionItemIndex } from '../collection/collection_widget.edit.strategy';
+import { getConvertedDataSource } from '@ts/ui/shared/m_grouped_data_converter_mixin';
```

Comment:
I suggest rename 'm_grouped_data_converter_mixin' to 'grouped.ts' and place it inside 'core/data/data_converter/' folder